liboptv/include/sortgrid.h

```diff
@@ -0,0 +1,39 @@
+/* Utilities for handling sortgrid */
+
+#ifndef SORTGRID_H
+#define SORTGRID_H
+
+#include "calibration.h"
+#include "parameters.h"
+#include "tracking_frame_buf.h"
+#include "imgcoord.h"
+#include "multimed.h"   
+#include "trafo.h"
+#include "vec_utils.h"
+#include <stdio.h>
+
+typedef struct
+{
+  int pnr;
+  vec3d pos;
+} coord_3d;
```

Comment:
I started this branch to adhere with your suggestion: https://github.com/alexlib/openptv/tree/remove_pnr_sortgrid

but i'm not sure now about the points are "numbered in the order of the file" - the calibration ponts in the calib.txt file we usually number using some arbitrary values (e.g. for multiplane we do with 100, 200, etc.) rather than the 'i' row value. how would we assign a pointer using these arbitrary id's ?